index.html

```diff
@@ -17,65 +22,85 @@
 			<li style="color:red"><a href="#uninstall-full" style="color:red">Uninstall SBPM</a></li>
 			<li><a href="#uninstall-part">Uninstall a command</a></li>
 			<li><a href="#update">Update the package manager</a></li>
-			<li><a href="#issues">Known issues</a></li>
+			<li><a href="#issues">Known issues</a></li></div>
 		</ul>
-		<hr/>
+		<div class="boxes">
 		<h3 id="install-local">To install commands pre-downloaded from other sources:</h3>
 		<ol>
 			<li>Move the downloaded commands into the <code>extensions</code> folder.</li>
 			<li>Double-click <code>install.bat</code></li>
 			<li>You're done! A log will be available at <code>logs\install.log</code>.</li>
 		</ol>
+		<div class="boxes">
 		<h3 id="install-global">To install commands available here:</h3>
 		<ol>
 			<li>Open Command Prompt (Win+R, type <code>cmd</code> and hit Enter/Win+X, C)</li>
 			<li>Type <code>sbpm install &lt;package></code> (replace <code>&lt;package></code> with the name of the desired package.)</li>
-			<li>You're done! A log will be available at <code>logs\install.log</code>.</li>
+			<li>You're done! A log will be available at <code>logs\install.log</code>.</li></div>
 		</ol>
-		<h3 id="upgrade-local">To upgrade commands pre-downloaded from other sources:</h3>
+				<div class="boxes">
+<h3 id="upgrade-local">To upgrade commands pre-downloaded from other sources:</h3>
 		<ol>
 			<li>Delete the previous versions from the <code>extension</code> folder.</li>
 			<li>Move the new versions into the <code>extension</code> folder.</li>
 			<li>Double-click <code>reinstall.bat</code></li>
 			<li>You're done! Logs will be available at <code>logs\uninstall.log</code> and <code>logs\install.log</code>.</li>
 		</ol>
+			</div>
+					<div class="boxes">
+
 		<h3 id="upgrade-global">To upgrade commands available here:</h3>
 		<ol>
 			<li>Open Command Prompt (Win+R, type <code>cmd</code> and hit Enter/Win+X, C)</li>
 			<li>Type <code>sbpm install &lt;package> /upgrade</code> (replace <code>&lt;package></code> with the name of the desired package.)</li>
 			<li>You're done! A log will be available at <code>logs\install.log</code>.</li>
-		</ol>
-		<h3 id="uninstall-full" style="color:red">To uninstall SBPM completely:</h3>
+						</ol></div>
+					<div class="boxes">
+
+		<h3 id="uninstall-full">To uninstall SBPM completely:</h3>
 		<ol>
 			<li>Double-click <code>uninstall.bat</code></li>
 			<li>You're done! A log will be available at <code>logs\uninstall.log</code>.</li>
 			<li>:(</li>
 		</ol>
+			</div>
+					<div class="boxes">
+
 		<h3 id="uninstall-part">To uninstall a command installed with SBPM:</h3>
 		<ol>
 			<li>Open Command Prompt (Win+R, type <code>cmd</code> and hit Enter/Win+X, C)</li>
 			<li>Type <code>sbpm uninstall &lt;package></code> (replace <code>&lt;package></code> with the name of the desired package.)</li>
 			<li>You're done! A log will be available at <code>logs\uninstall.log</code>.</li>
 		</ol>
+			</div>
+					<div class="boxes">
```

Comment:
This is some weird indentation - would you mind making everything consistent? I.e.
```html
    </ol>
</div>
<div class="boxes">
```